src/main/java/com/bandwidth/sqs/consumer/acknowledger/MessageAcknowledger.java

```diff
@@ -135,6 +141,10 @@ public Completable getCompletable() {
         return ackingComplete;
     }
 
+    public MessagePublisher<T> getMessagePublisher(){
```

Comment:
It makes it easy to wrap the Acknowledger in a different type. A JSON acknowledger could convert a JSON object to a normal SQS Message, this use this publisher to actually publish. (The could that would get the publisher doesn't have access to the publisher provided in the constructor)